edx_solutions_api_integration/courses/views.py

```diff
@@ -2199,75 +2171,44 @@ class CoursesMetricsSocial(MobileListAPIView):
     """
     ### The CoursesMetricsSocial view allows clients to query about the activity of all users in the
     forums
-    - URI: ```/api/users/{course_id}/metrics/social/?organization={org_id}```
-    - GET: Returns a list of social metrics for users in the specified course. Results can be filtered by organization
+    - URI: ```/api/courses/{course_id}/metrics/social/?organization={org_id}&score={bool}```
+    - GET: Returns a list of social metrics for users in the specified course.
+        * use `organization` query param to filter users by organization
+        * use `score` query param to get only scores of the users
```

Comment:
Sorry just notices, this should be 'scores', not 'score'.

Reply:
Maybe `users' scores` will be more accurate here?

Reply:
What I meant is, the actual parameter this API accepts is 'score**s**' not 'score'

Reply:
Oh, haha, right!